Src/FluentAssertions/Primitives/StringAssertions.cs

```diff
@@ -618,7 +618,7 @@ public AndConstraint<StringAssertions> NotEndWithEquivalentOf(string unexpected,
         /// <param name="becauseArgs">
         /// Zero or more objects to format using the placeholders in <see cref="because" />.
         /// </param>
-        public AndConstraint<StringAssertions> Contain(string expected, string because = "", params object[] becauseArgs)
+        public StringContainConstraint Contain(string expected, string because = "", params object[] becauseArgs)
```

Comment:
🔧 Would `StringContainmentConstraints` be a better name?

Reply:
Maybe `StringContainmentConstraint` (without 's' at the end)?

Reply:
I changed the name to `StringContainmentConstraint` because I think it better fits the convention e.g. `AndWhichConstraint`. If you really want to have `StringContainmentConstraints` I can change this.

Reply:
No, not at all. A 🔧 is a suggestion anyway.

Reply:
True, I know it is just a suggestion.
I didn't find any class with name ending with `Constraints`, so I thought it is convection.
Now I see I was wrong.
I changed class name to your suggestion :)